doc/releasenotes/14_0_0_summary.md

```diff
@@ -109,19 +128,15 @@ Online DDL is no longer experimental (with the exception of `pt-osc` strategy).
 - Revertible migrations
 - Declarative migrations
 - Postponed migrations
-- and all other functionality
+- And all other functionality
 
 Are all considered production-ready.
 
 `pt-osc` strategy (online DDL via 3rd party `pt-online-schema-change`) remains experimental.
 
-#### Throttling
-
-See new SQL syntax for controlling/viewing throttling for Online DDL, down below.
-
```

Comment:
@shlomi-noach this seemed to be duplicated below so I removed this section.
Can you please review all the changes in this PR to online DDL / throttling to make sure I didn't introduce errors in meaning / usage?

Reply:
All changes look good to me.